styles.css

```diff
@@ -3,3 +3,15 @@ body,
 div#__next {
     height: 100%;
 }
+
+div[data-overlay-container=true] {
+    height: 100%;
+}
+
+#background-container {
+    position: fixed;
+    left: 0;
+    top: 0;
+    width: 100vw;
+    height: 100vh
```

Comment:
Just a general warning that viewport sizes are - if I remember correctly - relative to the size of the device screen. Keep that in mind in regards to browser window size and special cases like navigation bar being hidden on mobile devices or a keyboard showing up. 

https://stackoverflow.com/questions/64176328/what-are-the-disadvantages-of-vh-and-vw-units

Reply:
yes, this class in particular is for the background. @i-am-the-slime proposed to have a separate div to render the background and give it height 100vh/vw. Seems to work. Is there a better way to do this?

Reply:
Was a general hint. I personally would more or less use flexbox, grid or `100%`.

If it works for the background then you're probably fine with it. But might find issues. E.g. scrolling on mobile seems to show one issue. Wouldn't overthink it now, but has to work with long lists etc.


https://user-images.githubusercontent.com/13085980/214128162-c6fd47aa-0f93-4626-8ce8-7cff77ab69aa.mp4

Reply:
yeah thats really bad. would be better to shrink this somehow than getting that scrollbar

Reply:
Yeah, but I was referring to the bottom. There you can see a white box appear during scrolling
 This is the effect of `100vh` I think.

Reply:
any idea how we can fix this?

Reply:
These things kill me. How do you get something that works in all circumstances without an army of testers?

Reply:
No. Army of testers is what you need ^^

I think fixing the navbar position, having the website container filling out the rest of the height and then only allow scrollable content within the container should work better. Right now the whole website can scroll.

Hate these things, too. I would merge it as is, fix it later, and focus on some progress with functionality. And as a general rule of thumb:  if possible, avoid any additional CSS with UI frameworks.